notes/interview.vbs

```diff
@@ -784,19 +784,19 @@ function define_main_dialog()
 			If disc_no_phone_number = "EXISTS" OR disc_no_phone_number = "RESOLVED" Then
 				GroupBox 10, y_pos, 455, 35, "No Phone Number, Review Phone Contact"
 				Text 20, y_pos + 20, 165, 10, "Confirm with the resident about phone contact."
-				ComboBox 185, y_pos + 15, 270, 45, "Select or Type"+chr(9)+"Confirmed No good phone contact"+chr(9)+"Added a Message Only Number"+chr(9)+"Added a Phone Number"+chr(9)+"Resident will Contact with a Phone Number once Obtained"+chr(9)+disc_phone_confirmation, disc_phone_confirmation
+				ComboBox 185, y_pos + 15, 270, 45, "Select or Type"+chr(9)+"Confirmed No good phone contact"+chr(9)+"Added a Message Only Number"+chr(9)+"Added a Phone Number"+chr(9)+"Resident will Contact with a Phone Number once Obtained"+chr(9)+"Resident uses free phone via internet program and pays no phone or internet bill"+chr(9)+disc_phone_confirmation, disc_phone_confirmation
```

Comment:
I think this one should not be updated, this box is talking about a number we can reach them at, not billing, and is covered with the existing "No good phone contact" option.

Reply:
Removed addition from this line